src/main/java/com/redhat/rhjmc/containerjfr/net/SslConfiguration.java

```diff
@@ -0,0 +1,207 @@
+package com.redhat.rhjmc.containerjfr.net;
+
+import com.redhat.rhjmc.containerjfr.core.log.Logger;
+import com.redhat.rhjmc.containerjfr.core.sys.Environment;
+import com.redhat.rhjmc.containerjfr.core.sys.FileSystem;
+import io.vertx.core.http.HttpServerOptions;
+import io.vertx.core.net.JksOptions;
+import io.vertx.core.net.PemKeyCertOptions;
+import io.vertx.core.net.PfxOptions;
+import org.openjdk.jmc.common.util.Pair;
```

Comment:
I'm trying to decouple container-jfr from direct dependencies on JMC classes[0]. Could you replace this Pair type with 1) a homegrown implementation (should be simple enough), 2) an Apache commons-lang Pair, 3) an abused AbstractMap.SimpleEntry, 4) a Map or some other structure

[0] some of this currently just means moving that dependency into container-jfr-core for potential independent reimplementation later

Reply:
Replaced with `org.apache.commons.lang3.tuple.Pair`